### ISSUE TITLE
Deleted Records not staying deleted after visiting Progress Page.  

### ISSUE THREAD

**Issue by kcostell** [Razviar/mtga-pro-tracker#349]
Using the "Account Wipe" option, I attempted to delete my record and match results (selecting all options except "Collection Progress".  The screen showed "Wipe complete", and the My Profile page showed a win/loss record of 0%.  

However, the "Progress" page still showed match results, and visiting the My Profile Page after the progress page then showed a nonzero win/loss record (the same record as before the wipe).

**Comment by Aquaman88** [comment]
Hi Thank you for your message. I can give you an advice to use the function "full reset" in your tracker. Also we know, that sometimes the information is updating during some time. You can try to turn CTRL+ F5 and check it again.

**Comment by kcostell** [comment]
This didn't seem to make a difference.  Steps I followed

1. Do a full reset
2. Do an account wipe on the website, selecting "Mathces (sic) results", "Events", "Drafts", "Matches", "Overall Matches Stats", "Reset Global W/L" 
3. Re-enter my token.  
4. Visit, in order, "My Profile Page", Events Page, Progress Page, My Profile Page, Matches List, and individual deck pages.  Ctrl-F5 each page.  

The first visit to the My Profile page showed a 0% W/L record (which I guess corresponds to how you're treating 0 wins/0 losses).  The Events and Matches page still showed events and matches (i.e. neither were wiped).  The Progress page showed match results.  The second visit to the My Profile page now showed a non-zero Lifetime W/L record (and not just the record from the matches recorded on the progress page).  Individual deck pages still showed several weeks worth of match history on them.

In short, neither the "Full Reset" nor the Account Wipe actually wiped most of the things I expected them to wipe.
